packages/angular/src/generators/web-worker/lib/normalize-options.ts

```diff
@@ -1,10 +1,41 @@
 import type { WebWorkerGeneratorOptions } from '../schema';
+import type { Tree } from '@nrwl/devkit';
+import {
+  getProjects,
+  joinPathFragments,
+  readProjectConfiguration,
+} from '@nrwl/devkit';
 
 export function normalizeOptions(
+  tree: Tree,
   options: WebWorkerGeneratorOptions
 ): WebWorkerGeneratorOptions {
+  if (!getProjects(tree).has(options.project)) {
+    throw new Error(`Project '${options.project} does not exist!`);
```

Comment:
Missing ending quote:
```suggestion
    throw new Error(`Project '${options.project}' does not exist!`);
```